public/js/report.js

```diff
@@ -742,6 +743,25 @@ var Hit = React.createClass({
 });
 
 /**
+ * Renders Length Visualisation of all hits per query
```

Comment:
I guess length visualisation could be of many type - length distribution would a more appropriate name.